modules/iam-profile/main.tf

```diff
@@ -0,0 +1,205 @@
+resource "aws_iam_role" "role" {
+  name = "${var.resource_prefix}-${var.aws_account_id}"
```

Comment:
based on prev comment, can this be changed to `integration_name + 8-hex-digits-of-hash(of account ID)`?
Whatever you decide, please inform the Siddharth